src/Service/CreateSnapshotService.php

```diff
@@ -45,6 +45,7 @@ public function createBySite(SiteInterface $site): void
     {
         $pages = $this->pageManager->findBy(['site' => $site->getId()]);
 
+        //I guess it's the problem
         if ($this->snapshotManager instanceof BaseEntityManager) {
```

Comment:
Hey @jordisala1991 do you remember why have you made this change?

https://github.com/sonata-project/SonataPageBundle/commit/f8022f6c39aa9eacfe8c055b21a4c45f666ae7e1#diff-ec7c09b3ae7003bdec9a47c70c05ea942e56535b0b83465513e0b1561e0ebdacR48

Reply:
I guess it's related with #1535 

https://github.com/sonata-project/SonataPageBundle/commit/f8022f6c39aa9eacfe8c055b21a4c45f666ae7e1#diff-112198c8b3816a6af7235c0c526ba39544d633700e4538d9ab1fc7e7c770d8f6R38

Reply:
What is the problem you have with snapshot creation? I have test the create snapshots command in a functional way, and they are created for sure. (Might be some edge case not covered by those test).

Reply:
https://github.com/sonata-project/SonataPageBundle/pull/1547/files#diff-ec7c09b3ae7003bdec9a47c70c05ea942e56535b0b83465513e0b1561e0ebdacR58

WOOPS, this should be an end transation, not sure how is called. ->commit() maybe?